feature/login/src/main/java/com/example/login/ui/screen/EmailVerificationScreen.kt

```
@@ -47,92 +47,84 @@ fun EmailVerificationScreen(
+    val emailValid = remember(email) {
+        Patterns.EMAIL_ADDRESS.matcher(email).matches()
     }
-    val context = LocalContext.current
-    val sendResult by viewModel.sendCodeResult.collectAsState()
-    val verifyResult by viewModel.verifyCodeResult.collectAsState()
-    val isCodeSent = sendResult == "인증 코드 전송 성공"
-    val isCodeValid = code.value.length == 6
+
+    val isCodeSent = authState is EmailAuthState.SendSuccess
+    val isCodeValid = code.length == 6
     val timerText = String.format("%02d:%02d", timer / 60, timer % 60)
```

detekt (1.23.8)

[warning] String.format("%02d:%02d", timer / 60, timer % 60) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)